mob_test.go

```diff
@@ -643,6 +643,37 @@ func TestResetDeleteRemoteWipBranchCommit(t *testing.T) {
 	assertNoMobSessionBranches(t, configuration, "mob-session")
 }
 
+func TestResetCommitBranch(t *testing.T) {
+	output, configuration := setup(t)
+	configuration.WipBranchQualifier = "green"
+
+	start(configuration)
+	createFile(t, "example.txt", "contentIrrelevant")
+	next(configuration)
+	assertMobSessionBranches(t, configuration, "mob/master-green")
+
+	reset(configuration)
+
+	assertOutputContains(t, output, "mob reset --delete-remote-wip-branch")
+	assertMobSessionBranches(t, configuration, "mob/master-green")
+}
+
+func TestResetDeleteRemoteWipBranchCommitBranch(t *testing.T) {
+	_, configuration := setup(t)
+	configuration.WipBranchQualifier = "green"
+	configuration.ResetDeleteRemoteWipBranch = true
```

Comment:
We could move this down, to the 'act' part, like we did in the other tests.
But I'm fine with this, too.

Reply:
That doesn't work here, as we need this configuration before we do mob start. I can duplicate it to have if in the act part to if you like.